.rubocop.yml

```diff
@@ -24,9 +24,24 @@ Metrics/BlockLength:
   Exclude:
     - spec/**/*
     - config/routes.rb
+    - app/models/neighbour_response.rb
 
 Metrics/MethodLength:
-  Max: 20
+  Max: 25
+
+Metric/AbcSize:
```

Comment:
I wonder if we should do what we do in BOPS itself and just disable the metrics cops:

```yaml
Metrics:
  Enabled: false
```